templates/public/download.html

```diff
@@ -68,8 +68,8 @@
 
     <p>If you have a wired connection, you can boot the latest release directly over the network.</p>
     <ul>
-        <li><a href="{{ releng_pxeboot_url }}"
-            title="Arch Linux Netboot Live System">Arch Linux Netboot</a></li>
+        <li><a href="/releng/netboot/"
+            title="Arch Linux Netboot">Arch Linux Netboot</a></li>
```

Comment:
Please use url tag and the name that you gave to that url

https://docs.djangoproject.com/es/1.9/ref/templates/builtins/#url